Puppetfile

```diff
@@ -18,6 +18,7 @@ mod 'puppetlabs-firewall',             '1.12.0'
 mod 'puppetlabs-hocon',                '1.0.0'
 mod 'puppetlabs-inifile',              '2.2.2' # Dependency of puppetlabs-puppetdb
 mod 'puppetlabs-kubernetes',           '3.1.0'
+mod 'puppetlabs-kubernetes', :git => 'https://github.com/fydai/puppetlabs-kubernetes'
```

Comment:
Should the line above this be removed?

Reply:
You made your commit off of master of puppetlabs-kubernetes in your fork, so by doing this you would be upgrading the version we are using to their master branch at the same time.

You may want to consider rebasing your changes off of version 3.1.0 in your fork, tagging it as something else, and using that as the version.